README.md

```diff
@@ -207,7 +217,7 @@ methods that are less secure that we will not go into). To use this method you w
 
 #### grunt link
 
-The link task is designed to ease the local development and testing of changes that span multiple packages. Traditionally `npm link` can be used but this assumes that the project structure is the same as the distribution, which for dojo2 projects is not the case. 
+The link task is designed to ease the local development and testing of changes that span multiple packages. Traditionally `npm link` can be used but this assumes that the project structure is the same as the distribution, which for dojo2 projects is not the case.
```

Comment:
dojo2 -> Dojo 2

Reply:
Didn't actually change this :joy: My editor just removed the trailing whitespace!